src/routes/(root)/sign-up/+page.server.js

```diff
@@ -0,0 +1,41 @@
+import { redirect } from '@sveltejs/kit';
```

Comment:
I think we can just combine this with signup by making it an action of a form. https://kit.svelte.dev/docs/form-actions then we don't have two seperate routes that are really the same.

Reply:
Or we make it a +server page and export it as a get https://kit.svelte.dev/docs/routing#server but I think I like the form approach better.

Reply:
We will need to know if the person is intending to login or sign-up so they get the correct form at Auth0 and, one thing I like about the sign-up now, is that the user is being redirected back to /my/profiles after signing up so they can fill in there details. Will this be easy enough to do from a from-action?

Reply:
but we have two signup routes